interfaces/console/pages/_app.tsx

```diff
@@ -11,23 +11,43 @@ import { commonData, isDarkmode, snackbarMessage } from '@/app-recoil';
 import client from '@/client/ApolloClient';
 import { theme } from '@/styles/theme';
 import { MyAppProps, TCommonData, TSnackMessage } from '@/types';
+import AuthWrapper from '@/ui/wrappers/authWrapper';
 import createEmotionCache from '@/ui/wrappers/createEmotionCache';
 import ErrorBoundary from '@/ui/wrappers/errorBoundary';
 import { ApolloProvider, HttpLink } from '@apollo/client';
 import { CacheProvider } from '@emotion/react';
 import { Alert, AlertColor, CssBaseline, Snackbar } from '@mui/material';
 import { ThemeProvider } from '@mui/material/styles';
 import dynamic from 'next/dynamic';
+import { useEffect } from 'react';
 import { RecoilRoot, useRecoilState, useRecoilValue } from 'recoil';
 import '../styles/global.css';
-
 const MainApp = dynamic(() => import('@/pages/_main_app'));
 const clientSideEmotionCache = createEmotionCache();
 const SNACKBAR_TIMEOUT = 5000;
 
+const getMetaInfo = async () => {
+  return await fetch('https://api.ipify.org/?format=json', {
```

Comment:
Can you move  this api url in constant ?

Reply:
Addressed